### PR TITLE
basic preact example

### PR DESCRIPTION
**Pull request by philipodev**
Basic, bare minimum, preact example with working hmr.

preact-example

### CODE REVIEW

File: preact/package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "parcel-react-example",
```
Comment (brandon93s) [inline preact/package.json]: "react"

File: preact/package.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  "version": "1.0.0",
+  "description": "Parcel Bundler Example with React",
```
Comment (brandon93s) [inline preact/package.json]: "react"

**Comment by philipodev** [comment]
@brandon93s oh, my bad, was to quick. this has now been fixed. thank you.

**Comment by brandon93s** [comment]
Looks great (Win 10)! HMR works nicely, too.

**Pull request merged**